execution-engine-commons/src/test/java/com/odysseusinc/arachne/execution_engine_common/ConnectionParamsParserTest.java

```diff
@@ -0,0 +1,14 @@
+package com.odysseusinc.arachne.execution_engine_common;
+
+import com.odysseusinc.arachne.commons.types.DBMSType;
+import com.odysseusinc.arachne.execution_engine_common.util.ConnectionParamsParser;
+import org.junit.Test;
+
+public class ConnectionParamsParserTest {
+
+    @Test
+    public void testParsingParamWithEmptyValue() {
+
+        ConnectionParamsParser.parse(DBMSType.POSTGRESQL, "jdbc:postgresql://localhost:64290/postgres?binaryTransferEnable=&unknownLength=2147483647");
```

Comment:
I don't see any asserts.

Reply:
What assert would you expect to see here? The thing is that this method previously crashed with JDBC URL params w/o value (remove changes from `ConnectionParamsParser.java` and try to run test - you'll see exception). But those param-value pairs are not saved into the DTO produced by the method. So, really, nothing to check - either it runs or nor.

Reply:
I got why you did this, but it does not really matter what was before. Right now `testParsingParamWithEmptyValue` test is confusing,  as a developer, I am looking at it, and cannot see what is the purpose of this test(and I don't want to spend time to figure it out). So I would recommend to get the result of parsing and check that `binaryTransferEnable=""` and `unknownLength=2147483647`